src/types/index.d.ts

```diff
@@ -18,3 +18,4 @@ export * from './request-executor';
 export * from './default-request-executor';
 export * from './collection';
 export * from './generated';
+export * from './memory-store';
```

Comment:
@denysoblohin-okta This is an auto-generated file. I believe this change should go into the `templates/index.d.ts.hbs` file. I added that to my PR, so the build script won't remove this line: https://github.com/okta/okta-sdk-nodejs/pull/377.